scripts/actions/mobskills/smite_of_rage.lua

```diff
@@ -10,11 +10,14 @@ end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local numhits = 1
-    local accmod = 1
-    local dmgmod = 2.7
+    local accmod  = 1
+    local dmgmod  = mob:getWeaponDmg() * 0.03
```

Comment:
This is already being calculated inside the `mobPhysicalMove`

dmgmod is basically a flat FTP right now the way its coded. 

It will hit this: `local base = math.max(1, mob:getWeaponDmg() + dSTR)` which the damage will return `level + 2` which will then calculate the hitdamage after it with dlvl here:

```
--work out the base damage for a single hit
local hitdamage = math.max(1, base + lvldiff) * dmgMod
```

It appears the only thing that needs to change in this PR would be the 2.7 to 3 as per FTP and shadows to 1 per retail testing shown here: https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=57955395

Reply:
yep #5062 baked in the multiplier for us. older scripts need an audit soon to prevent copy pasting `mob:getWeaponDmg()` into new ones